.github/workflows/tests-pytest.yml

```diff
@@ -31,9 +26,6 @@ jobs:
       - name: Install Python dependencies
         run: pip install -e .[test]
 
-      - name: Run setup
-        run: ./bin/init.sh
-
```

Comment:
See failed action job logs here: https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/actions/runs/14541702445/job/40800834021